test/tests/functional/pbs_multi_sched.py

```diff
@@ -1322,3 +1322,37 @@ def test_list_multi_sched(self):
             err_msg = "Unknown Scheduler"
             self.assertTrue(err_msg in e.msg[0],
                             "Error message is not expected")
+
+    def test_job_sort_formula_threshold(self):
+        """
+        Test the scheduler attribute job_sort_formula_threshold for multisched
+        """
+        # Multisched setup
+        self.setup_sc3()
+        a = {'queue_type': 'execution',
+             'started': 'True',
+             'enabled': 'True'}
+        self.server.manager(MGR_CMD_CREATE, QUEUE, a, id='wq1')
+        p3 = {'partition': 'P3'}
+        self.server.manager(MGR_CMD_SET, QUEUE, p3, id='wq1', expect=True)
```

Comment:
add partition P3 to the dict while creating the queue itself

Reply:
@visheshh  I have made the changes.